modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics using FlowMetrics API
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
+. Use the *Project:*  dropdown list to select the project of the Network Observability Operator instance. 
+. Click *Create FlowMetric*.
+. Here are a couple of examples of `FlowMetrics` configurations you can add:
++
+.Generate a metric that tracks ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+  namespace: netobserv                              <1>
+spec:
+  metricName: cluster_external_ingress_bytes_total  <2>
+  type: Counter                                     <3>
+  valueField: Bytes
+  direction: Ingress                                <4>
+  labels: [DstK8S_HostName,DstK8S_Namespace,DstK8S_OwnerName,DstK8S_OwnerType] <5>
+  filters:                                          <6>
+  - field: SrcSubnetLabel
+    matchType: Absence
+----
+<1> The same namespace, `netobserv`, is used across all example components in this documentation. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
+<2> How the metrics appear, along with the prefix `netobserv`. 
+<3> The `type` `Counter` is useful for counting bytes or packets. Histogram is the other available type, which is useful for latencies. 
+<4> Direction of traffic to capture. Egress is an alternative option. If not specified, both ingress and egress are captured, which can lead to duplicated counts. 
+<5> Labels define what the metrics look like and the relationship between the different entities. The labels also define the metrics cardinality. For example, `SrcK8S_Name` is a high cardinality metric.
```

Comment:
May be rephrase to combine both into a single sentence. I do not have the technical context hence not sure if my suggestion would be accurate. Also, one question: Is cardinality a word that would need an explanation?

Reply:
On cardinality, I can see that it's used in other parts of the documentation, without much explanation. [This page](https://docs.openshift.com/container-platform/4.15/support/troubleshooting/investigating-monitoring-issues.html#determining-why-prometheus-is-consuming-disk-space_investigating-monitoring-issues) mentions cardinality and shows a procedure for debugging high cardinality issues. I don't know, perhaps that would be good to point to that page for troubleshooting cardinality ?

Reply:
Okay I can link to this. I think thats a good idea, and good to point out that cardinality may not be widely discussed in the docs. Thanks both :)